style.css

```diff
@@ -0,0 +1,93 @@
+header {
+  margin-top: 20px;
+}
+
+.navigator {
+  margin-top: 10px;
+  padding: 20px;
+  display: flex;
+  background-color: white;
+  justify-content: flex-end;
+  margin-right: 10px;
+  align-items: center;
+}
+
+.mylogo {
+  display: flex;
+  flex-grow: 2;
+  width: 76px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  align-items: center;
+  color: #6070ff;
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
+}
+
+.hamburger {
+  display: flex;
+  flex-grow: 1;
+}
+
+.container {
+  background-image: url('images/Header bg.png');
+  display: flex;
+  flex-wrap: wrap;
+  flex-basis: 100px;
+  flex-direction: column;
+  padding-left: 5px;
+}
```

Comment:
<html>
<body>
<!--StartFragment--><p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">
I see you tried to implement the change requested here by the previous reviewer but it's not done quite right. KIndly check on your background positioning in this section.
</p>

Figma design | Your design
-- | --
![image](https://user-images.githubusercontent.com/29071984/170827529-794cc2c0-a5da-4401-877b-32fb3ed667c3.png) |  ![image](https://user-images.githubusercontent.com/29071984/170827553-d65ba6a0-e933-43e1-be2c-11750729f404.png)


<!--EndFragment-->
</body>
</html>